lib/expandable_page_view.dart

```diff
@@ -82,9 +82,21 @@ class _ExpandablePageViewState extends State<ExpandablePageView> {
   bool _shouldDisposePageController = false;
   bool _firstPageLoaded = false;
 
-  double get _currentHeight => _heights.contains(_currentPage) ? _heights[_currentPage] : _heights.first;
+  double get _currentHeight {
+    try {
+      return _heights[_currentPage];
+    } catch (_) {
+      return _heights.first;
```

Comment:
powinno się spr. czy _currentPage jest w zakresie, a nie catchowac error.

Reply:
myślałem nad tym, ale to co proponujesz dokładnie? czasem _currentPage może być np -3

Reply:
Poszedł update